docker-compose-generator/docker-fragments/opt-addnodes.yml

```diff
@@ -0,0 +1,26 @@
+version: "3"
+# Uses addnodes from Prohashing explorers for faster syncing! 
+
+services:
+  dogecoind:
+    environment:
+      DOGECOIN_EXTRA_ARGS: |
+        addnode=coins.prohashing.com:6003
+  feathercoind:
+    environment:
+      BITCOIN_EXTRA_ARGS: |
+        addnode=coins.prohashing.com:6002
+  litecoind:
+    environment:
+      BITCOIN_EXTRA_ARGS: |
+        addnode=coins.prohashing.com:6001
+  viacoind:
+    environment:
+      BITCOIN_EXTRA_ARGS: |
+        addnode=coins.prohashing.com:6095
+  monacoind:
+    environment:
+      BITCOIN_EXTRA_ARGS: |
+        addnode=coins.prohashing.com:6138
+exclusive:
```

Comment:
Remove this as there is no reason it would be exclusive since a node can have multiple addnode config items

Reply:
Users should be able to add "addnodes" - depending on the use case.  Prohashing is a reliable source that runs seeder nodes, and I will be happy to document.

Reply:
I was referring to the `exclusive` section :)

Reply:
I thought exclusive was used when ignoring fragments, like this ```BTCPAYGEN_EXCLUDE_FRAGMENTS```
What is that section used for then?

Reply:
It is used to avoid conflicting fragments, for example someone including a pruning fragment with the txindex fragment